packages/ag-charts-website/src/pages/[framework]/[pageName].astro

```diff
@@ -48,7 +48,7 @@ const descriptionWithSeoTagline = `${description} ${seoTagline}`;
 
 <link rel="preconnect" href="https://fonts.googleapis.com" />
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-<link href="https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500&display=swap" rel="stylesheet" />
+<link href="https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500;700&display=swap" rel="stylesheet" />
```

Comment:
Be good to check with KP exactly what font weights we need. Do we have bold monospace text anywhere? Do we need 400 AND 500?

Reply:
So `500` is not used, so have removed it